vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/java/com/vaadin/flow/component/shared/HasPrefixAndSuffix.java

```diff
@@ -13,26 +13,25 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
-package com.vaadin.flow.component.textfield;
```

Comment:
Probably a minor "breaking change" that should be noted somewhere :) It's kinda unrelated to the tabsheet component, so it's possible that this can be missed later on.

Reply:
Good point. @tomivirkki Could you please extract this to the separate PR?

Reply:
Added. the PR https://github.com/vaadin/flow-components/pull/3714

...but I'm also considering just copy-pasting the `HasPrefixAndSuffix` methods directly to TabSheet to avoid breaking any add-ons that may be using the interface

Reply:
Done, ended up not relocating the interface at this point since it is a breaking change.

Reply:
I'm wondering if it would be worth it to still create the interface in the base project and deprecating the old on in the TextField package? This could also be done with a `HasSuffix` and `HasPrefix` interface, allowing e.g. other components to use either of them (like Combobox, where only Prefix is supported)

Reply:
It could probably use some API design, so leaving it out of the scope of this PR

Reply:
Of course :) Totally out of scope! Still thanks for looking into the breaking change thing.